test/jdk/javax/management/remote/nonLocalAccess/NonLocalJMXRemoteTest.java

```diff
@@ -154,4 +222,43 @@ private static void assertIsAccessException(Throwable ex) {
             throw new RuntimeException("AccessException did not occur when expected", ex);
```

Comment:
private static void assertIsAccessException(Throwable ex) is define in all these three test classes, move this method in lib/jdk/test/lib

Reply:
This method is test dependent that checks for specific keywords in the AccessException. It seems better remain inside the test classes.